### ISSUE TITLE
Merge Rows causes issue

### ISSUE THREAD

**Issue by kfred** [stephenliberty/excel-builder.js#45]
Hi,

Merging columns works fine for me, but when I merge rows, e.g. (A1,A3) I get recovery message when opening the file in Excel. No data loss or data corruption occurs, but I would like to fix the cause of this issue and row merging is necessary for my project.

Has anyone encountered this?

**Comment by kfred** [comment]
More specifically, this issue only occurs when I programmatically apply the merged cells. If I enter fixed values, the mergeCells across rows function works fine. But when I create dynamic values, it causes an error:

I have tried .concat to add the dynamic number to the cell column letter, and have also tried applying .toString() to my dynamically generated row number, but none of these solutions appear to resolve the "repair" message that Excel displays.

```
       for (var i = 0; i < mergedRows.length; i++) {
            if (rows < 1) { continue; }
            var rowN = Number(mergedRows[i][0]), rows = Number(mergedRows[i][1]);
            var endRow = rowN + rows;
            console.log("Row number: " + rowN + " & " + rows + " rows");
            var cola = "G", colb = "H", colc = "K", cold = "L";
            workbook.mergeCells(cola.concat(rowN),cola.concat(endRow));
            workbook.mergeCells(colb.concat(rowN),colb.concat(endRow));
            workbook.mergeCells(colc.concat(rowN),colc.concat(endRow));
            workbook.mergeCells(cold.concat(rowN),cold.concat(endRow));
        }
```
Any thoughts?

**Comment by kfred** [comment]
Okay, well, by spending the time to write this out this issue, I was able to trace it back my code. This is not a bug and can be ignored.

My mergedRows array referred to above contained duplicate entries; once I removed those duplicate entries/values, the mergeCells function worked as expected.

However, as a heads up to others who may be dynamically applying the mergeCells function to your workbook, you will get a recovery message in Excel if you apply the a mergeCells function twice to the same set of cells.
